locales/en-US/app.ftl

```diff
@@ -122,6 +122,11 @@ retirePageHeadline = Thanks for flying!
 retirePageMessage = Hope you had fun experimenting with us. <br> Come back any time.
 retirePageSurveyButton = Take a quick survey
 
+restartIntroLead = Preflight checklist
```

Comment:
These are different than the strings in the component. They should probably be the same.